### DISCUSSION TITLE
Multiple coordinators/networks

### DISCUSSION THREAD

**Discussion by nottledim** [Koenkk/zigbee2mqtt#5751] [General]
I want to control 2 distinct zigbee networks from home assistant and was wondering if there was a way to this with zigbee2mqtt.

One could either have 2 coordinators managed by one zigbee2mqtt server or have 2 servers with one coordinator each presenting devices to home assistant.  

Are either of these possibilities at the moment?

**Comment by Koenkk** [comment]
1 coordinator = 1 zigbee2mqtt instance. For multiple coordinators you need to run multiple zigbee2mqtt instances. Make sure to configure a different `base_topic` for each: https://www.zigbee2mqtt.io/information/configuration.html

**Comment by nottledim** [comment]
Thanks, that's what I thought but I wasn't sure how home-assistant could discover it.  How does that work?  (I have to configure mosquitto to pass topics). The instances will be on different computers.

**Comment by nottledim** [comment]
I think I've got it now.  Discovery is by device sending message to topic homeassistant/.  Much appreciated.
